common.gypi

```diff
@@ -29,7 +29,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.12',
+    'v8_embedder_string': '-node.13',
```

Comment:
Just a question, is this required, just a formality, or does it help keep track of patches?

Reply:
Required by the V8 update guide. I would be +1 on either removing the requirement for `deps/v8/gypfiles` or moving the directory to another place.

Reply:
> or moving the directory to another place.

I want to do that, I just need to find the time.